fv3config/cli.py

```diff
@@ -0,0 +1,26 @@
+import argparse
+
+import yaml
+import fsspec
+
+import fv3config
+
+
+def _parse_args():
+    parser = argparse.ArgumentParser("write_run_directory")
+    parser.add_argument(
+        "config", help="URI to fv3config yaml file. Supports any path used by fsspec."
```

Comment:
Will Daint be able to access a remote fv3config.yaml file? Should work if we are running interactively on a login or data-mover node of Piz Daint.

Reply:
It should work if GOOGLE_APPLICATION_CREDENTIALS are set correctly, but I think that's a prerequisite for using fv3config overall.